update note (#1374)
codemagic-ci-cd/codemagic-docs: content/rest-api/artifacts.md

## Retrieve public download URL

> {{<notebox>}}
> **Important!** Please take extra care when sharing public download URLs so as to not expose them. Using a public download URL, anyone will be able to access the build artifacts.
> {{</notebox>}}
> 

Reviewer: ```suggestion
{{<notebox>}}
**Important!** Please take extra care when sharing public download URLs so as to not expose them. Anyone with access to your public download URL will be able to download your build artifact.
{{</notebox>}}

```
